module/VuFind/src/VuFind/Marc/MarcReader.php

```diff
@@ -277,4 +277,161 @@ public function getFieldsSubfields(string $fieldTag, array $subfieldCodes,
 
         return $result;
     }
+
+    /**
+     * Return an associative array for a linked field such as 880 (Alternate Graphic
+     * Representation) or an empty array if field does not exist
+     *
+     * @param string $fieldTag       The MARC field that contains the linked fields
+     * @param string $linkedFieldTag The linked MARC field tag to get
+     * @param string $occurrence     The occurrence number to get; empty string for
+     * whatever comes first
+     * @param array  $subfieldCodes  The MARC subfield codes to get, or empty for all
+     *
+     * @return array
+     */
+    public function getLinkedField(string $fieldTag, string $linkedFieldTag,
+        string $occurrence = '', ?array $subfieldCodes = null
+    ): array {
+        $results
+            = $this->getLinkedFields($fieldTag, $linkedFieldTag, $subfieldCodes);
+        if ('' === $occurrence) {
+            return $results[0] ?? [];
+        }
+        foreach ($results as $field) {
+            if ($occurrence === $field['link']['occurrence']) {
```

Comment:
If you just add `|| empty($occurrence)` here, you can eliminate the '' special case above and get the same behavior... though perhaps it is less clear. Just a thought!